apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -935,3 +997,23 @@ async function listAllObjectKeys(bucket: R2Bucket, prefix: string): Promise<stri
 
 	return keys
 }
+
+function* generateSnapshotChunks(snapshot: RoomSnapshot, clock: number): Generator<Uint8Array> {
+	const encoder = new TextEncoder()
+
+	yield encoder.encode(`{"clock":${clock},"tombstones":`)
```

Comment:
Why can't we use the snapshot clock here?

Reply:
yeah should be fine to. We should also include the new property `tombstoneHistoryStartsAtClock` that was added in #6488

Reply:
Sounds good, just added that. 

Feels like this change might make it a bit prone to future updates, maybe we should improve the chunk generation to be more generic and pull things out of the snapshot instead of hardcoding the values.

Reply:
yeah sounds smart, go for it.